src/QirRuntime/test/QIR-static/qir-driver.cpp

```diff
@@ -58,7 +58,7 @@ extern "C" int64_t Microsoft__Quantum__Testing__QIR__Test_Arrays( // NOLINT
     int64_t index,
     int64_t val,
     bool dummy);
-TEST_CASE("QIR: Using 1D arrays", "[qir]")

```

Comment:
In general, style guide is to avoid the term "dummy," as that can be an exclusive term; something like "placeholder" or even "_" is strongly preferred.

Reply:
I would prefer to refrain dealing with this case of "dummy" in this PR since "dummy" is out of this PR's scope. This case of "dummy" has important functionality behind it (not knowing which can result in a loss of a few hours for a new-comer), and if I touch this "dummy" I feel obliged to document in details that important functionality.

However I'm inclined to agree about the use of "dummy" in general, and would like to know more about it and other similar identifiers/names. Any direct link about more info? (preferably reachable from [Coding Style](https://ms-quantum.visualstudio.com/Quantum%20Program/_wiki/wikis/Quantum-OKRs.wiki/487/Coding-Style))